### ISSUE TITLE
How to get training data? Why is the betas parameter 200?

### ISSUE THREAD

**Issue by Timkeeper2018** [YadiraF/PIXIE#41]
As you mentioned in your paper, the training data is SMPL-X fit of exPose. But the number of betas parameters given by exPose is 10, but the number of betas parameters given in your method is 200, I was wondering how you guys Get the SMPL-X dataset with betas parameter of 200? (I see that most of the methods set the number of betas parameter of SMPL-X to 10)

**Comment by MichaelJBlack** [comment]
Indeed, most people use 10, but this gives quite a coarse shape.  If one is fitting to 2D joint locations, 10 is enough because there is very little information about body shape in a few 2D joints.  The AMASS dataset uses 16.  We make 300 available for download.  People can choose how many they need for their application -- with 300, you can get very high-fidelity body shape but it is hard to reliably estimate 300 from images.

**Comment by Timkeeper2018** [comment]
Wait, does that mean you have a training dataset with betas of 300? However, I did not find the download option of the relevant training dataset on your project page, so kind of you if you can share the training dataset. If it is not convenient for public disclosure, this is my email: hackerclaymore@gmail.com.  
Thanks a lot
